src/client.js

```diff
@@ -270,25 +271,43 @@ Client.prototype.input = function(data) {
 	var client = this;
 	var text = data.text.trim();
 	var target = client.find(data.target);
+
 	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+		text = "say " + text;
+	} else {
+		text = text.substr(1);
+
+		// If after removing first / we still have a /, send it as a message
+		if (text.charAt(0) === "/") {
+			text = "say " + text;
+		}
 	}
+
 	var args = text.split(" ");
-	var cmd = args.shift().replace("/", "").toLowerCase();
+	var cmd = args.shift().toLowerCase();
+	var result;
+
 	_.each(inputs, function(plugin) {
```

Comment:
Instead of `_.each`, use the [native `.some()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).
You can then `return fn.apply(...)` instead of `result = fn.apply(...)`, and have `var result = inputs.some(...)`.
Makes it cleaner and you can drop `return !result;` as well as your comment R300-302 :-)